client/dashboard/domains/domain-connection-setup/style.scss

```diff
@@ -0,0 +1,27 @@
+@import "@wordpress/base-styles/colors";
+@import "@wordpress/base-styles/variables";
+
+.dashboard-domain-connection-verification {
```

Comment:
I see we also have a `domain-connection-verifications.scss` file. Should these styles be here or in that file? 

Additionally. the `domain-connection-verification.tsx` file imports `style.scss`. I would expect the root `index.tsx` file to import that stylesheet.